index.bs

```diff
@@ -5820,34 +5822,66 @@ Note: In order to interoperate, user agents storing large blobs on authenticator
 :: `largeBlob`
 
 : Operation applicability
-:: [=authentication extension|Authentication=]
+:: [=registration extension|Registration=] and [=authentication extension|authentication=]
 
 : Client extension input
 ::  <xmp class="idl">
     partial dictionary AuthenticationExtensionsClientInputs {
         AuthenticationExtensionsLargeBlobInputs largeBlob;
     };
 
+    enum LargeBlobSupport {
+      "required",
+      "preferred",
+    };
+
     dictionary AuthenticationExtensionsLargeBlobInputs {
+        DOMString support;
         boolean read;
         ArrayBuffer write;
```

Comment:
Semi unrelated to this PR, but should this be a `BufferSource` instead? That's what we're using elsewhere for byte sequences.

Reply:
Yep! Thanks. Done.